src/localization.js

```diff
@@ -262,7 +262,7 @@ angular.module('ngLocalize')
             return currentLocale;
         }
 
-        setLocale(cookieStore ? cookieStore.get(localeConf.cookieName) : $window.navigator.userLanguage || $window.navigator.language);
+        setLocale(cookieStore.get(localeConf.cookieName) ? cookieStore.get(localeConf.cookieName) : $window.navigator.userLanguage || $window.navigator.language);
```

Comment:
How would cookieStore evaluate to `false` and `cookieStore.get(localeConf.cookieName)` evaluate to `true`? If there is `cookieStore` is false then `cookieStore.get(localeConf.cookieName)` will definitely be `false` as well. Seems like an unnecessary check. Am I overlooking something?